cmake/external.cmake

```diff
@@ -117,6 +117,9 @@ function(link_libabsl target_name)
         endif()
     endif()
     if (NOT absl_FOUND)
+        if (NOT TARGET libabsl)
```

Comment:
Why other cases don't do this check?

Reply:
Only link_libabsl called twice. Will I add this check to other cases?

Reply:
In theory any of them could be called any amount of times